### PR TITLE
[Documentation][Gpr_To_Absl_Logging] Fix GRPC_VERBOSITY docs

### PR DESCRIPTION
**Pull request by tanvi-jagtap**
[Documentation][Gpr_To_Absl_Logging] Fix GRPC_VERBOSITY docs

### CODE REVIEW

File: TROUBLESHOOTING.md [hunk 1]
```diff
@@ -11,11 +11,14 @@ that gets printed to stderr.
...
 <!-- BEGIN_GOOGLE_INTERNAL_DOCUMENTATION
 GRPC_VERBOSITY has been disabled for internal usage and will not work anymore.
-If anyone wants to debug, we need to set verbose logs using absl.
+If anyone wants to debug, we need to [set log verbosity using absl](https://abseil.io/docs/cpp/guides/logging).
+
 END_GOOGLE_INTERNAL_DOCUMENTATION -->
 
 <!-- BEGIN_OPEN_SOURCE_DOCUMENTATION -->
-`GRPC_VERBOSITY` is used to set the minimum level of log messages printed by gRPC (supported values are `DEBUG`, `INFO` and `ERROR`). If this environment variable is unset, only `ERROR` logs will be printed. `ERROR` is recomeded for production systems.
+Our recommendation is to avoid using this flag and [set log verbosity using absl](https://abseil.io/docs/cpp/guides/logging). We only support this flag for legacy reasons.
```
Comment (yashykt) [inline TROUBLESHOOTING.md:19]: `GRPC_VERBOSITY` should not be a top level section anymore. It doesn't make sense to start off by saying "Don't do this". It should instead say - "Do this".

And then, as a side-note, you can talk about the legacy support for `GRPC_VERBOSITY`.

File: doc/environment_variables.md [hunk 2]
```diff
@@ -53,7 +53,10 @@ If anyone wants to debug, we need to set verbose logs using absl.
 END_GOOGLE_INTERNAL_DOCUMENTATION -->
 
 <!-- BEGIN_OPEN_SOURCE_DOCUMENTATION -->
-`GRPC_VERBOSITY` is used to set the minimum level of log messages printed by gRPC (supported values are `DEBUG`, `INFO` and `ERROR`). If this environment variable is unset, only `ERROR` logs will be printed.
+  `GRPC_VERBOSITY` is used to set the minimum level of log messages printed. Supported values are `DEBUG`, `INFO` and `ERROR`.
```
Comment (yashykt) [inline doc/environment_variables.md:56]: On line 49, let's add `(DEPRECATED)`

Comment (tanvi-jagtap) [reply doc/environment_variables.md:51]: Done

File: TROUBLESHOOTING.md (hunk 1 above)
Comment (tanvi-jagtap) [reply TROUBLESHOOTING.md]: Changed everything.

**Review by yashykt — APPROVED** [review]

**Pull request closed without merging**